packages/vulcan-ui-material/lib/components/core/Datatable.jsx

```diff
@@ -132,14 +134,16 @@ class Datatable extends PureComponent {
         />
       );
     } else {
-      const { className, collection, options, showSearch, showNew, classes } = this.props;
+      const { className, options, showSearch, showNew, classes } = this.props;
+
+      const collection = this.props.collection || getCollection(this.props.collectionName);
 
       const listOptions = {
         collection: collection,
         ...options,
       };
 
-      const DatatableWithMulti = withMulti(listOptions)(Components.DatatableContents);
+      const DatatableWithMulti = compose(withMulti(listOptions))(Components.DatatableContents);
```

Comment:
Why the compose call hre?

Reply:
I saw it here: https://github.com/VulcanJS/Vulcan/blob/devel/packages/vulcan-core/lib/modules/components/Datatable/Datatable.jsx#L219 so I decided to keep similar code style to the Datatable in core.